src/DigitalCard.js

```diff
@@ -13,6 +14,7 @@ export default function DigitalCard() {
             <div className="main">
                 <About />
                 <Interests />
```

Comment:
Please update Interests as "Food expert. Music scholar. Reader. Internet fanatic. Bacon buff. Entrepreneur. Travel geek. Pop culture ninja. Coffee fanatic." might not be up to date